metrics/integration/render-tests/text-max-width/zero-width-line-placement/style.json

```diff
@@ -0,0 +1,66 @@
+{
+  "version": 8,
+  "metadata": {
+    "test": {
+      "height": 128,
+      "width": 512
+    }
+  },
+  "center": [
+    10,
+    0
+  ],
+  "zoom": 4,
+  "sources": {
+    "mapbox": {
```

Comment:
Consider renaming this to maplibre or geojson

Reply:
I copied these from MapLibre GL JS, will you also change them there? 

Otherwise it will show up as a disparity when I update my tool to read the contents of these files.

Reply:
Sure, why not. Or you could create a PR with the updated file.

Reply:
OK!